packages/terra-demographics-banner/package.json

```diff
@@ -21,7 +21,8 @@
   },
   "homepage": "https://github.com/cerner/terra-core#readme",
   "devDependencies": {
-    "terra-toolkit": "^0.4.0"
+    "terra-toolkit": "^0.4.0",
+    "react-intl": "^2.3.0"
```

Comment:
Curious why react-intl needs to be added as a devDependency, is this for testing?